src/main/scala/slack/models/package.scala

```diff
@@ -155,6 +167,7 @@ package object models {
           case "channel_history_changed" => JsSuccess(jsValue.as[ChannelHistoryChanged])
           case "im_created" => JsSuccess(jsValue.as[ImCreated])
           case "im_opened" => JsSuccess(jsValue.as[ImOpened])
```

Comment:
I wasn't sure whether it was safe to remove this line, as the message comes in as `im_open`. Let me know and I'll clean it up.